src/collection.js

```diff
@@ -623,6 +623,9 @@ export default class Collection {
     return this.get(id, {includeDeleted: true})
       .then(res => {
         const existing = res.data;
+        if (existing._status === "deleted") {
+          throw new Error(`Record with id=${id} not found.`);
+        }
```

Comment:
I think we could just get rid of the `includeDeleted` option in the call and maybe even remove it from `get`.